### PR TITLE
content: more commodities

### PR DESCRIPTION
**Pull request by NRK4**
## Summary
I added a lot of commodities in multiple categories. The categories with the most additions are Clothing, Food, and Medical, but there are some new ones in several other categories as well.

### CODE REVIEW

File: data/commodities.txt [hunk 1]
```diff
@@ -103,51 +116,115 @@ trade
...
 		"socks"
+		"suits"
+		"sundresses"
 		"sunglasses"
 		"surgical scrubs"
+		"suspenders"
+		"sweaters"
+		"sweatpants"
 		"swimsuits"
+		"tanktops"
```
Comment (tibetiroka) [inline data/commodities.txt:206]:
```suggestion
		"tank tops"
```

File: data/commodities.txt [hunk 2]
```diff
@@ -318,39 +395,60 @@ trade
...
 		"prosthetic limbs"
+		"reflex hammers"
 		"rubbing alcohol"
 		"scanners"
+		"speculums"
+		"stethoscopes"
 		"surgical equipment"
 		"surgical robots"
+		"tissues"
+		"tongue depressers"
```
Comment (tibetiroka) [inline data/commodities.txt:447]:
```suggestion
		"tongue depressors"
```

File: data/commodities.txt [hunk 3]
```diff
@@ -430,6 +528,7 @@ trade
 		"capacitors"
 		"circuit boards"
 		"cloud chambers"
+		"conic cegrameters"
```
Comment (tibetiroka) [inline data/commodities.txt:531]:
```suggestion
```
I'm not sure what this is supposed to be, but I couldn't find anything similar.

File: data/commodities.txt [hunk 4]
```diff
@@ -480,6 +580,7 @@ trade
 		"touchscreens"
 		"trackpads"
 		"transceivers"
+		"tradometers"
```
Comment (tibetiroka) [inline data/commodities.txt:583]:
```suggestion
```
Once again, I'm clueless.

File: data/commodities.txt [hunk 5]
```diff
@@ -606,14 +711,19 @@ trade
 		"carbon steel"
 		"composite beams"
 		"composite fibers"
+		"concrete"
+		"duck tape"
```
Comment (tibetiroka) [inline data/commodities.txt:715]:
```suggestion
		"duct tape"
```

File: data/commodities.txt [hunk 6]
```diff
@@ -632,6 +742,7 @@ trade
 		"hallucinogens"
 		"horse tranquilizers"
 		"memory chips"
+		"star shrooms"
```
Comment (tibetiroka) [inline data/commodities.txt:745]:
```suggestion
```
No idea here either.

**Review by tibetiroka — CHANGES_REQUESTED** [review]
Should be pretty good in terms of grammar.

`colposcopes` and `teffs` were not found in some dictionaries, but the words themselves are correct. `eye charts`, `otoscopes` and `ophthalmoscopes` were missing from even more, but are also fine.

Also, are you sure `pirated software` fits in as a commodity? `stolen military secrets` might work, but I don't think pirated software would fit in the same category.

File: data/commodities.txt (hunk 5 above)
Comment (NRK4) [reply data/commodities.txt:715]: good catch

File: data/commodities.txt (hunk 2 above)
Comment (NRK4) [reply data/commodities.txt:447]: committed

File: data/commodities.txt (hunk 3 above)
Comment (NRK4) [reply data/commodities.txt:531]: It's just something I made up. In the 31st century there is no doubt a lot of new technology that we do not currently have, so it wouldn't make sense to have only items from the 21st century. My philosophy is that ideally there should be a mix of current and futuristic technologies to add plausibility that the game takes place in the future while also providing familiarity for the player. A couple of made up items are already in the file, such as a couple of creative names in the illegal substances category. I decided to try my hand at creating some in other categories. If people don't like them it is alright with me if they aren't included, but I would like to hear people's opinions.

File: data/commodities.txt (hunk 4 above)
Comment (NRK4) [reply data/commodities.txt:583]: See above

File: data/commodities.txt (hunk 6 above)
Comment (NRK4) [reply data/commodities.txt:745]: See "conic cegrameters" above

File: data/commodities.txt (hunk 1 above)
Comment (NRK4) [reply data/commodities.txt]: committed

**Comment by NRK4** [comment]
You have a point with pirated software. Teff is a crop grown primarily in Ethiopia for its seeds. I wanted to diversify the food list a bit, and part of that in my mind meant incorporating more foods from around the globe, which is also why I added sorghum. All of the medical instruments are real as far as I could tell based on my research, although as a disclaimer I am not a medical professional.

File: data/commodities.txt (hunk 3 above)
Comment (roadrunner56) [reply data/commodities.txt:531]: I personally like having stuff that doesn't exist (yet). It adds to the feel that ES society is substantially different from today's society, which I don't feel like there's much of. It just feels like our current civilization but in space. Transporting tech that's unique to the 31st century helps mitigate that a bit.

**Review by roadrunner56 — APPROVED** [review]
Looks good to me.

File: data/commodities.txt [hunk 7]
```diff
@@ -103,51 +116,115 @@ trade
...
 		"carpeting"
 		"children's clothing"
+		"clogs"
 		"coats"
+		"costumes"
 		"cotton sheets"
 		"cotton thread"
 		"diapers"
 		"denim"
+		"diapers"
```
Comment (W1zrad) [inline data/commodities.txt:146]:
```suggestion
		"denim"
		"diapers"
```

File: data/commodities.txt [hunk 8]
```diff
@@ -103,51 +116,115 @@ trade
...
 		"umbrellas"
 		"underwear"
 		"uniforms"
+		"wallets"
+		"watches"
+		"wedding dresses"
+		"wetsuits"
 		"winter coats"
 		"women's shoes"
 		"waterproof fiber"
```
Comment (W1zrad) [inline data/commodities.txt:223]:
```suggestion
		"wallets"
		"watches"
		"waterproof fiber"
		"wedding dresses"
		"wetsuits"
		"winter coats"
		"women's shoes"
```

**Review by W1zrad — COMMENTED** [review]
Nice! Noticed a few existing errors adjacent to your new stuff that I'll fix up real quick.

**Pull request merged**